libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthRootController.h

```diff
@@ -31,10 +31,6 @@
  */
 
 #import "SFSDKRootController.h"
-
-#if __IPHONE_OS_VERSION_MIN_REQUIRED >= 130000
+#import <AuthenticationServices/AuthenticationServices.h>
 @interface SFSDKAuthRootController : SFSDKRootController <ASWebAuthenticationPresentationContextProviding>
```

Comment:
Does this build on iOS 12?
Before the if def, I had a cast in the @available link but no reference to iOS 13 classes otherwise.

Reply:
Yes I was able to build it for for iOS 12.

Reply:
And no runtime issue?

Reply:
Compiles and runs in both.

Reply:
How come the compiler does not complain? ASWebAuthenticationPresentationContextProviding is only available on iOS 13.